src/screens/ChatDetailPage/index.js

```diff
@@ -123,7 +124,6 @@ const ChatDetailPage = ({route}) => {
   }, []);
   React.useEffect(() => {
     searchUserMessages(channelClient.channel?.cid);
-    setParticipants(channelClient.channel?.state?.members, dispatch);
   }, [clients.client]);
   const searchUserMessages = async (channelID) => {
     const messages = await clients.client.search(
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the new hunk, the `setParticipants` function call has been removed. This function was previously used to set the participants of the chat. If this functionality is handled elsewhere or no longer needed, then this change is fine. Otherwise, this could lead to unexpected behavior where the participants are not properly set.
```diff
-    setParticipants(channelClient.channel?.state?.members, dispatch);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->